### PR TITLE
feat(suite): add phase countdown to the critical phases modal

### PR DESCRIPTION
**Pull request by dahaca**
## Description

Added a phase countdown to the critical phases modal

## Related Issue

follow up for https://github.com/trezor/trezor-suite/issues/6117

## Screenshots:

image

### CODE REVIEW

File: packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/PhaseProgress.tsx [hunk 1]
```diff
@@ -68,9 +77,10 @@ const Step = ({ phase, currentPhase }: StepProps) => {
 
 interface PhaseProgress {
```
Comment (komret) [inline packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/PhaseProgress.tsx]: PhaseProgressProps?

File: packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/PhaseProgress.tsx [hunk 2]
```diff
@@ -68,9 +77,10 @@ const Step = ({ phase, currentPhase }: StepProps) => {
...
     currentPhase: RoundPhase;
+    phaseDeadline: number | string;
```
Comment (komret) [inline packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/PhaseProgress.tsx]: Can this really be a string? It seems the type can be narrowed down in `CoinjoinSession` interface and also here to just `number`.

File: packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/PhaseProgress.tsx [hunk 3]
```diff
@@ -84,5 +94,11 @@ export const PhaseProgress = ({ currentPhase }: PhaseProgress) => (
             <Translation id={COINJOIN_PHASE_MESSAGES[currentPhase]} />
             ...
         </Message>
+
+        {phaseDeadline && (
```
Comment (komret) [inline packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/PhaseProgress.tsx]: The condition seems redundant.

**Review by komret — CHANGES_REQUESTED** [review]

File: packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/PhaseProgress.tsx (hunk 2 above)
Comment (dahaca) [reply packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/PhaseProgress.tsx]: I briefly discussed it with Szymon and it was decided to keep it like that :)

File: packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/PhaseProgress.tsx (hunk 3 above)
Comment (dahaca) [reply packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/PhaseProgress.tsx]: dcd11dc6f606fcbde9a674b4050ce92b690ab9c3

File: packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/PhaseProgress.tsx (hunk 1 above)
Comment (dahaca) [reply packages/suite/src/components/suite/modals/CriticalCoinjoinPhase/PhaseProgress.tsx]: dcd11dc6f606fcbde9a674b4050ce92b690ab9c3

**Review by komret — APPROVED** [review]

**Pull request merged**